docker/run_tests.sh

```diff
@@ -34,6 +35,7 @@ echo ">>> wait for services"
 urlwait "${DATABASE_URL}" 10
 urlwait "http://${PUBSUB_EMULATOR_HOST}" 10
 urlwait "${ELASTICSEARCH_URL}" 10
+urlwait "${CRASHSTORAGE_ENDPOINT_URL}" 10
```

Comment:
Previously, all the boto/s3 things were mocked out with moto or other things. This PR ditches moto in favor of talking to the localstack-s3 service directly. These changes make sure that service is running.